spec/app_templates/config/initializers/clearance.rb

```diff
@@ -0,0 +1,3 @@
+Clearance.configure do |config|
+  config.mailer_sender = "reply@example.com"
```

Comment:
Yes, this is a template used by the generator specs. It's the pre-existing initializer for this test case.